src/Materials/Textures/babylon.texture.ts

```diff
@@ -619,8 +619,17 @@ module BABYLON {
 
                     return mirrorTexture;
                 } else if (parsedTexture.isRenderTarget) {
-                    var renderTargetTexture = new RenderTargetTexture(parsedTexture.name, parsedTexture.renderTargetSize, scene, generateMipMaps);
-                    renderTargetTexture._waitingRenderList = parsedTexture.renderList;
+                    let renderTargetTexture: Nullable<RenderTargetTexture> = null;
+                    if (parsedTexture.isCube) {
+                        const probe = ReflectionProbe.Parse(parsedTexture, scene, rootUrl);

```

Comment:
Do we still need that? I'm a bit worried because we could save a cube without using reflection probe

Reply:
Right,
So I'm now looking for an existing reflection probe instead of creating it
Other cube textures should be created/found if no reflection probe found
Is that ok for you?